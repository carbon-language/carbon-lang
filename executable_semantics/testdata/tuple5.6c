// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

// Test the that field order matters for tuples.

fn main() -> Int {
  var (.x = Int, .y = Int) t = (.y = 2, .x = 3);
  return 0;
}
