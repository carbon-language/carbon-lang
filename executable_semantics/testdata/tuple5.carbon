// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

// Test the that field order matters for tuples.

fn main() -> i32 {
  var t: (.x = i32, .y = i32) = (.y = 2, .x = 3);
  return 0;
}
