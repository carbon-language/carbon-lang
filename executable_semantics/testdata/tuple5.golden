8: type error in pattern variable
expected: (x = Int@2, y = Int@3)
actual: (y = Int@0, x = Int@1)
EXIT CODE: 255
